src/bindings/python/tests/test_graph/test_custom_op.py

```diff
@@ -197,3 +199,89 @@ def test_custom_op():
     input_tensor = Tensor(input_data)
     results = request.infer({"data": input_tensor})
     assert np.allclose(results[list(results)[0]], expected_output, 1e-4, 1e-4)
+
+
+class CustomSimpleOp(Op):
+    def __init__(self, inputs=None):
+        super().__init__(self, inputs)
+
+    def validate_and_infer_types(self):
+        self.set_output_type(0, self.get_input_element_type(0), self.get_input_partial_shape(0))
+
+
+class CustomSimpleOpWithAttribute(Op):
+    class_type_info = DiscreteTypeInfo("CustomSimpleOpWithAttribute", "extension")
+
+    def __init__(self, inputs=None, **attrs):
+        super().__init__(self, inputs)
```

Comment:
Does `super().__init__` call `validate_and_infer_types` in this case?

Reply:
Yes, it does

Reply:
It calls `construct_and_validate_infer_types`, which calls `validate_and_infer_types`. So yes.